css/css-align/gaps/gap-normal-computed-001.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Box Alignment Test: computed value of normal on *-gap properties</title>
+<link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#column-row-gap">
```

Comment:
I believe the policy is somewhat fuzzy, but that linking to TR when the text we want is on TR is preferred, and that linking to the ED otherwise is OK. I wouldn't lose sleep over it :)